content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>nit: in the sentence " <code>training_labels</code>: labels corresponding to the training images of size 60,000." the reader might get the impression that the images are of size 60000. I'd change it to something like </p><p> </p><p><code>training_labels</code>:  array of size 60,000 corresponding to the labels of the training images</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Nice catch.

```diff
 - `training_images`: flattened training images of size 60,000x784.
- - `training_labels`: labels corresponding to the training images of size 60,000.
+ - `training_labels`: array of size 60,000 corresponding to the labels of the training set.
 - `test_images`: flattened test images of size 60,000x784.
- - `test_labels`: labels corresponding to the training images of size 60,000.
+ - `test_labels`: array of size 10,000 corresponding to the labels of the test set.
```